internal/commands/root/sign.go

```diff
@@ -78,12 +78,16 @@ func commandSign(o *options, s *gsio.Streams, args ...string) error {
 		return fmt.Errorf("failed to create rekor client: %w", err)
 	}
 
-	sig, cert, tlog, err := git.Sign(ctx, rekor, userIdent, dataBuf.Bytes(), signature.SignOptions{
+	opts := signature.SignOptions{
 		Detached:           o.FlagDetachedSignature,
 		TimestampAuthority: o.Config.TimestampURL,
 		Armor:              o.FlagArmor,
 		IncludeCerts:       o.FlagIncludeCerts,
-	})
+	}
+	if o.Config.RekorMode == "offline" {
+		opts.RekorAddr = o.Config.Rekor
+	}
+	sig, cert, tlog, err := git.Sign(ctx, rekor, userIdent, dataBuf.Bytes(), opts)
```

Comment:
Nit: `tlog` is kinda confusing as a name, maybe `tlogEntry` (same elsewhere too)

Reply:
I know that inscrutability is Idiomatic Go (TM) so you can fight me on this